circuit-overlay/src/jvmTest/kotlin/com/slack/circuit/overlay/OverlayTest.kt

```diff
@@ -48,7 +48,7 @@ class OverlayTest {
     moleculeFlow(RecompositionMode.Immediate) {
         val overlayHost = rememberOverlayHost()
         val overlayHostData by rememberUpdatedState(overlayHost.currentOverlayData)
-        key(overlayHostData) { overlayHostData?.let { data -> data.overlay.Content(data::finish) } }
+        key(overlayHost) { overlayHostData?.let { data -> data.overlay.Content(data::finish) } }
```

Comment:
what was the impetus for this one?

Reply:
Was getting stuck in a recomposition loop as the data was changed. Side effect of the more unique key changes with 1.7